tests/unit/store/settings.spec.js

```diff
@@ -172,15 +172,15 @@ describe("store/settings", () => {
         user_ids: array_of_userid,
       },
     ];
-    expect(store.state.settings.customer_account_ids.length).toEqual(0);
+    expect(store.state.settings.customer_account_ids).toHaveLength(0);
     store.commit("settings/set_customer_account_ids", array_of_customerids);
     store.commit("settings/set_customer_index", 0);
     store.commit("settings/set_user_index", 0);
-    expect(
-      store.getters["settings/user_ids_for_selected_customer_account"].length
-    ).toEqual(3);
+    expect(store.state.settings.customer_account_ids[0].user_ids).toHaveLength(
+      3
+    );
   });
-  test("Store a multiple customer detail with multiple user ids can be assigned to the customer_ids array", () => {
+  test("When initialized the array of customer_account_ids is empty and size 0, Then commit an array of customer details in customer_account_ids and assert the number of customer records", () => {
```

Comment:
similar comment as previous test

Reply:
Would like to retain the present title, its more informative.